README.md

```diff
@@ -49,6 +50,15 @@ _A guide to writing and maintaining good markup._
 Set your editor to use spaces, not tabs.
 
 
+<a name="stick-to-a-line-length-of-100-characters"></a>
+### Stick to a line length of 100 characters.
+
+* Wrap lines that are overlong.
+* Align attribute names for all lines after the first.
+* Indent wrapped lines at least twice to distinguish them from nested elements and honor the previous item.
```

Comment:
I've been trying out an HTMLTidy linter in Sublime for a few months now...I just need to figure out what settings we want (and there are a lot!)